python/ray/autoscaler/_private/aws/config.py

```diff
@@ -33,28 +33,33 @@
 DEFAULT_RAY_IAM_ROLE = RAY + "-v1"
 SECURITY_GROUP_TEMPLATE = RAY + "-{}"
 
-# V61.0 has CUDA 11.2
-DEFAULT_AMI_NAME = "AWS Deep Learning AMI (Ubuntu 18.04) V61.0"
```

Comment:
@0x2b3bfa0 Actually, (sorry for realizing this late) why not use https://aws.amazon.com/releasenotes/aws-deep-learning-gpu-ami-ubuntu-20-04-version-2-x-x/ ? We can use the version with CUDA 11.2 to minimize any potential disruption.

Reply:
Found it from this list: https://docs.aws.amazon.com/dlami/latest/devguide/appendix-ami-release-notes.html#appendix-ami-release-notes-base

Reply:
(It may be that your choice is the correct one, just want to hear your thoughts)

Reply:
Good idea! Sorry for the terribly late reply! 🙈

Reply:
Resolved with https://github.com/ray-project/ray/pull/39721/commits/2df6cda89a06b6d311e3971e91631ea34adaf6ff